test/es2017/asyncFunctions.js

```diff
@@ -304,7 +304,7 @@ module.exports = function () {
     it('should handle async functions in autoInject', (done) => {
         async.autoInject({
             z: async function(){ return 0},
```

Comment:
nit: add a space after `function` to match the style elsewhere (yah, I know you didn't make this change, but it sticks out after you made the change to remove "a " in the next line)